UIElementsSchema/UIElements.xsd

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's the purpose of this whole `UIElementsSchema`? It wasn't here before and everything worked anyway, so I'm not quite sure what does it do.

Also **if it's not necessary**, and Unity is generating it, perhaps it should be added to `.gitignore`.

Reply:
Unity is generating it. After end of sprint I check this if is necessary ;)